src/components/CommentReplyPreview/CommentReplyPreview.tsx

```diff
@@ -98,30 +113,53 @@ export const CommentReplyPreview = ({
         </span>
       </div>
       {displayReplyComment && (
-        <div className={previewStyle}>
-          <p className={replyPreviewHeaderStyle}>
-            {commentBeingRepliedTo.userProfile.displayName} @{" "}
-            {commentBeingRepliedTo.date} said:
-          </p>
-          <p
-            dangerouslySetInnerHTML={{
-              __html: commentBeingRepliedTo.body || ""
-            }}
-          />
-          <div>
-            <span
-              className={cx(
-                hideCommentButtonStyles,
-                commentControlsButtonStyles,
-                previewHideCommentButtonStyles
-              )}
-              onClick={() => setDisplayReplyComment(!displayReplyComment)}
-            >
-              {displayReplyComment ? "Hide Comment" : "Show comment"}
-            </span>
-          </div>
-        </div>
+        <Preview
+          pillar={pillar}
+          commentBeingRepliedTo={commentBeingRepliedTo}
+          setDisplayReplyComment={setDisplayReplyComment}
+          displayReplyComment={displayReplyComment}
+        />
       )}
     </>
   );
 };
+
+export const Preview = ({
+  commentBeingRepliedTo,
+  pillar,
+  setDisplayReplyComment,
+  displayReplyComment
+}: {
+  commentBeingRepliedTo: CommentType;
+  pillar: Pillar;
+  setDisplayReplyComment: (displayReplyComment: boolean) => void;
+  displayReplyComment: boolean;
+}) => {
+  const commentControlsButtonStyles = commentControlsButton(pillar);
+  return (
+    <div className={previewStyle}>
+      <p className={replyPreviewHeaderStyle}>
+        {commentBeingRepliedTo.userProfile.displayName} @{" "}
+        {commentBeingRepliedTo.date} said:
+      </p>
+      <div
+        className={commentStyles}
+        dangerouslySetInnerHTML={{
+          __html: commentBeingRepliedTo.body || ""
+        }}
+      />
+      <div>
+        <span
```

Comment:
This should be a Button from source

Reply:
I tried but it assumes the background is white, so we get a strange  effect
<img width="237" alt="Screenshot 2020-04-02 at 22 12 16" src="https://user-images.githubusercontent.com/8831403/78300248-184f8a00-752f-11ea-8517-5b603f198358.png">

I decided to keep with a basic HTML button instead. We should likely raise the issue in src-foundations